js/sectionView.js

```diff
@@ -8,6 +8,7 @@ viewSection.handleMainNav = function(){
     $('.main-section').each(function(index){
```

Comment:
You don't need a loop here. You can select an id with just $('#idname'), where idname is the name of the id. You can concatenate strings in JS. You really don't need a loop.